modules/govuk/manifests/apps/email_alert_service.pp

```diff
@@ -51,6 +51,21 @@
     command            => './bin/email_alert_service',
   }
 
+
+  govuk::procfile::worker { 'email-alert-service-major-change-consumer':
+    setenv_as      => $app_name,
+    enable_service => $enable_major-change-queue-consumer,
```

Comment:
I think this will need to use only underscores in the variable name, and you'll also need to add it to the list of variables near the top around line 32-38.